pv.js

```diff
@@ -63,7 +66,13 @@ class Pv {
             this.pvLine = null;
         }
     }
-
+    postPvToChatFree(errline) {
+        const stop = this.STOPRE.exec(errline);
+        if (stop && this.pvLine) {
```

Comment:
How does this work? `this.pvLine` is always false for me.

Reply:
it doesn't work, I made a draft PR to discuss

Reply:
If you remove the check for `&& this.pvLine`, or if you set `this.pvLine` to a non-falsy value, like done in `postPvToChatSingleLine`, it will work.